backgroundsOptional/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./index.css" />
+    <title>Optional</title>
+</head>
+<body>
+    <div class="forest">
+        <div class="text">
+            <h1>I am Jane Doe</h1>
+            <h4>And I'm a Photographer</h4>
+            <button>Read more</button>
+        </div>
+    </div>
+    <div class="overlay"></div>
```

Comment:
As sugera ca overlay sa fie pus in interiorul la forest.
Astfel, width-ul si height-ul sa fie calculat in functie de dimensiunea parintelui peste care dorim sa punem acest overlay